### PR TITLE
fixed stringify settings

### PR DESCRIPTION
**Pull request by rg2011**
Fix issue #763

### CODE REVIEW

**Comment by AlvaroVega** [comment]
CNR update is needed

**Comment by rg2011** [comment]
> CNR update is needed

fix in 1da27e2963ef1bc5fa7fdd9d9472aec2105a8359

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1 +1 @@
-
+- Do not stringify template in postAction if it is already a string (issue #763)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: CNR-style

```suggestion
- Fix: do not stringify template in postAction if it is already a string (#763)
```

**Review by AlvaroVega — APPROVED** [review]
LGTM

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**